cpp/thirdparty/versions.txt

```diff
@@ -25,35 +25,39 @@
 
 # Abseil LTS branch, Sept 2020, Patch 2
 ARROW_ABSL_BUILD_VERSION=0f3bb466b868b523cf1dc9b2aaaed65c77b28862
-ARROW_AWSSDK_BUILD_VERSION=1.8.90
+ARROW_AWSSDK_BUILD_VERSION=1.8.128
+# aws-checksums 0.1.10 has some issues it seems
 ARROW_AWS_CHECKSUMS_BUILD_VERSION=v0.1.5
 ARROW_AWS_C_COMMON_BUILD_VERSION=v0.4.59
 ARROW_AWS_C_EVENT_STREAM_BUILD_VERSION=v0.1.5
 ARROW_BOOST_BUILD_VERSION=1.71.0
-ARROW_BROTLI_BUILD_VERSION=v1.0.7
+ARROW_BROTLI_BUILD_VERSION=v1.0.9
 ARROW_BZIP2_BUILD_VERSION=1.0.8
-ARROW_CARES_BUILD_VERSION=1.16.1
+ARROW_CARES_BUILD_VERSION=1.17.1
 ARROW_GBENCHMARK_BUILD_VERSION=v1.5.2
 ARROW_GFLAGS_BUILD_VERSION=v2.2.2
 ARROW_GLOG_BUILD_VERSION=v0.4.0
 ARROW_GRPC_BUILD_VERSION=v1.33.2
```

Comment:
1.34.1 and 1.35.0 are out.

Reply:
ARROW-11363 suggests that 1.34 won't work for us yet.

Reply:
I don't know, Kou's comment on that issue seems to suggest otherwise.

Reply:
I think that we can use the latest gRPC by 8cb2d81dc33f921974f8ef779a42d35a61ad195f.
If it doesn't work, could you show error message?